core/cap-0050.md

```diff
@@ -0,0 +1,183 @@
+```
+CAP: 0050
+Title: Smart Contract Signatures
+Working Group:
+    Owner: Jonathan Jove <@jonjove>
+    Authors:
+    Consulted:
+Status: Draft
+Created: 2022-05-10
+Discussion: TBD
+Protocol version: TBD
+```
+
+## Simple Summary
+
+This proposal explains how to authorize a smart contract interaction.
+
+## Motivation
+
+Actions in smart contracts must be authorized by an external entity. The
+authorization method can vary by smart contract.
+
+Basic authorization provides the authority to interact with a smart contract in
+the name of an account. For example, in CAP-48 and CAP-49 this level of
+authorization is sufficient to transfer funds.
+
+A more sophisticated aspect of authorization is the ability for one account to
+grant authority to another account or contract. In some cases, it is sufficient
+for the grantor to take an on-chain action. For example, in CAP-48 and CAP-49
+`approve`, which depends on basic authorization, can be used to grant control
+over the grantor's funds. In other cases, it is preferable for the grantee to
+take the on-chain action. This is demonstrated by `permit` in EIP-2612. When the
+grantee takes the on-chain action, the grantor must have presigned a message
+which conveys the relevant authority.
+
+This proposal is concerned with basic authorization and presigned messages.
```

Comment:
What is a presigned message, exactly? Just any payload that was signed? How is that different than... just a signed message? Just trying to unravel this for myself